changelog/3149.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Correctly implement the image warping when transforming Maps for differential
```

Comment:
Change to "Implement the image warping when transforming Maps for differential".  This is needed if we are saying the code is experimental.